app/src/main/java/edu/uc/reedws/musiclink/IndividualPlaylistActivity.kt

```diff
@@ -19,8 +19,8 @@ class IndividualPlaylistActivity : AppCompatActivity() {
         shareButton.setOnClickListener {
             val intent = Intent(Intent.ACTION_SEND)
             intent.type = "type/plain"
-            intent.putExtra(Intent.EXTRA_TEXT, "This is example text.")
-            intent.putExtra(Intent.EXTRA_SUBJECT, "Example Subject")
+            intent.putExtra(Intent.EXTRA_TEXT, "One of your friends shared a playlist with you through MusicLink!")
```

Comment:
As mentioned in my pull request, this is just a recommendation that I think may improve the functionality of the program/app. Just might be a cool thing to see, and I think that it could improve the code in general by being a little more specific to the user and have some sort of template besides the "example text"

Reply:
As long as you are changing this, take it one step further and move the string to strings.xml.  

strings.xml is the best place for string literals.  You can have a different strings.xml for each language you support, and the Android operating system will automatically choose the correct file, based on the user's preferred language.